### PR TITLE
Add labels.jq endpoint.

### PR DESCRIPTION
**Pull request by gislenius**

### CODE REVIEW

File: queries/modules/io/28/modules/xbrl/components.jq [hunk 1]
```diff
@@ -91,6 +111,33 @@ declare function components:components($component-or-ids as item*) as object*
...
+  $roles as string*,
+  $concepts as string*,
+  $exact-labels as string*,
+  $options as object*
+) as object*
+{
+  let $aids := if(deep-equal($archives-or-ids, $components:ALL-ARCHIVES))
+               then $components:ALL-ARCHIVES
+               else archives:aid($archives-or-ids)
+  let $query := {|
```
Comment (dknochen) [inline queries/modules/io/28/modules/xbrl/components.jq]: could we throw an error if all of the parameters are set to $components:ALL-*?

File: queries/modules/io/28/modules/xbrl/components.jq [hunk 2]
```diff
@@ -56,6 +56,26 @@ declare variable $components:ROLE as xs:string := "Role";
 declare variable $components:DEFAULT-LANGUAGE as xs:string := "Language";
 
 (:~
+ : Placeholder for all archive IDs..
+ :)
+declare variable $components:ALL-ARCHIVES as xs:string := "";
```
Comment (dknochen) [inline queries/modules/io/28/modules/xbrl/components.jq]: I really think we should not use an empty string here. It is hard to see the semantics of it when you debug. Could we use "ALL" instead? That would be even more consistent.

**Comment by dknochen** [comment]
@gislenius 2 simple comments only. Thank you!

File: queries/modules/io/28/modules/xbrl/components.jq (hunk 1 above)
Comment (gislenius) [inline queries/modules/io/28/modules/xbrl/components.jq]: DONE

File: queries/modules/io/28/modules/xbrl/components.jq (hunk 2 above)
Comment (gislenius) [inline queries/modules/io/28/modules/xbrl/components.jq]: DONE

File: queries/modules/io/28/modules/xbrl/components.jq [hunk 3]
```diff
@@ -91,6 +111,43 @@ declare function components:components($component-or-ids as item*) as object*
...
+  |}
+  return
+    switch(true)
+    case $roles eq $components:ALL-ROLES and
+         $aids eq $components:ALL-ARCHIVES and
+         $concepts eq $components:ALL-CONCEPTS and
+         $exact-labels eq $components:ALL-LABELS
+      return error(
+        QName("components:TWO-MANY-COMPONENTS"),
+        "Two many components to be returned because no filtering is done."
```
Comment (dknochen) [inline queries/modules/io/28/modules/xbrl/components.jq]: typo: two -> too and TWO -> TOO ;)

Comment (gislenius) [inline queries/modules/io/28/modules/xbrl/components.jq]: DONE ;-)

**Comment by wcandillon** [comment]
This is so cool. Can I redeploy edinet?

**Pull request merged**